docs/development/submitting-patches.rst

```diff
@@ -8,8 +8,6 @@ Submitting patches
   fork is also nice because that will you can work on more than one PR at a
   time.
 * Patches should be small to facilitate easier review.
-* New features and significant bug fixes should be documented in the
-  :doc:`/changelog`.
```

Comment:
The changelog seems to be empty, so I don't see that this rule is being followed.
And I didn't see a changelog in https://pypi.python.org/pypi/mimic